integrations/tensorflow/cpp_projects/dynamic_images/dynamic_images_test.cc

```diff
@@ -0,0 +1,280 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <array>
+#include <chrono>
+#include <cstdint>
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"
+#include "cpp_projects/dynamic_images/detection_post_process.h"
+#include "tensorflow/core/platform/logging.h"
+#include "tensorflow/lite/c/common.h"
+#include "tensorflow/lite/interpreter.h"
+#include "tensorflow/lite/kernels/register.h"
+#include "tensorflow/lite/model.h"
+#include "tensorflow/lite/model_builder.h"
+#include "tensorflow/lite/string_util.h"
+#include "tensorflow/lite/tools/evaluation/stages/image_preprocessing_stage.h"
+#include "tensorflow_lite_support/examples/task/vision/desktop/utils/image_utils.h"
```

Comment:
Nope nope nope. Let's not take a dependency on TensorFlow's C++ (even TFLite) for a sample, at least in the core IREE repo. https://github.com/google/iree-samples _maybe_, but this isn't IREE - it's TensorFlow.